src/main/webapp/js/userMapData.json

```diff
@@ -0,0 +1,826 @@
+["ROC van Amsterdam","Netherlands"],
+["RoCvA","Netherlands"],
+["Rotterdam University of Applied Science","Netherlands"],
+["Saxion","Netherlands"],
+["Saxion University","Netherlands"],
+["The Hague University of Applied Sciences","Netherlands"],
+["TIAS Business School","Netherlands"],
+["TU Delft","Netherlands"],
+["University of Inholland","Netherlands"],
+["University of Twente","Netherlands"],
+["University Utrecht","Netherlands"],
+["Utrecht University","Netherlands"],
+["Windesheim Honours College","Netherlands"],
+["Unitec Institute of Technology","New Zealand"],
+["University of Auckland","New Zealand"],
+["University of Waikato","New Zealand"],
+["NTNU","Norway"],
+["Institute of Business Management","Pakistan"],
+["National University of Sciences and Technology","Pakistan"],
+["Open-Quechua","Peru"],
+["Universidad de Ingeniería y Tecnología","Peru"],
+["Universidad Nacional de San Agustín","Peru"],
+["Asian Institute of Management","Philippines"],
+["Ateneo de Manila University","Philippines"],
+["Ateneo School of Medicine & Public Health","Philippines"],
+["Batangas State University","Philippines"],
+["De La Salle University","Philippines"],
+["EKindling","Philippines"],
+["Holy Cross of Davao College","Philippines"],
+["La Salle University","Philippines"],
+["Miriam College High School","Philippines"],
+["University of Manila","Philippines"],
+["University of the Cordilleras","Philippines"],
+["University of the Philippines","Philippines"],
+["Xavier School","Philippines"],
+["Adam Mickiewicz University","Poland"],
+["Universidade do Minho","Portugal"],
+["Universidad del Este","Puerto Rico"],
+["Carnegie Mellon University","Qatar"],
+["Qatar University","Qatar"],
+["SUNY Korea","Republic of Korea"],
+["Deltacredit","Russia"],
+["National Research University","Russia"],
+["NRU-Higher School of Economics","Russia"],
+["Edinburgh Napier School of Eng. and the Built Env.","Scotland"],
+["University of St Andrews","Scotland"],
+["Bedok Green Secondary","Singapore"],
+["Embry-Riddle Aeronautical University","Singapore"],
+["Experimental Systems and Technologies Lab","Singapore"],
+["Hwa Chong Institution","Singapore"],
+["Institute of Technical Education","Singapore"],
+["International Community School","Singapore"],
+["ITE","Singapore"],
+["Methodist Girls&#39; School","Singapore"],
+["Millennia Institute","Singapore"],
+["NAFA","Singapore"],
+["Nanyang Junior College","Singapore"],
+["Nanyang Technological University","Singapore"],
+["National University of Singapore","Singapore"],
+["Ngee Ann Polytechnic","Singapore"],
+["Punggol Green Primary School","Singapore"],
+["Raffles College of Higher Education","Singapore"],
+["Raffles Junior College","Singapore"],
+["Sengkang Secondary School","Singapore"],
+["SHATEC Pte Ltd","Singapore"],
+["Silicon Straits","Singapore"],
+["Singapore Chinese Girls' School","Singapore"],
+["Singapore Institute of Technology","Singapore"],
+["Singapore Management University","Singapore"],
```

Comment:
On on slightly unrelated note, how do we deal with the problem of Singapore being too small to show up in the map?